### PR TITLE
Send pushes to subscribers

### PR DESCRIPTION
**Pull request by jh0ker**
:sparkles:

### CODE REVIEW

File: entrypoint/fb.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 const facebook = require('../lib/facebook');
 const dialogflow = require('dialogflow');
 const handler = require('../handler');
+const request = require('request-promise-native');
+
+const pushes_url = `${process.env.CMS_API_URL}pushes/`;
```
Comment (mraerino) [inline entrypoint/fb.js]: There is a global `urls.js` for this stuff

File: entrypoint/fb.js [hunk 2]
```diff
@@ -106,3 +110,64 @@ module.exports.message = (event, context, callback) => {
...
 };
+
+module.exports.push = (event, context, callback = console.log) => {
+  let params = null;
+
+  if (event.queryStringParameters !== undefined) {
+    params = event.queryStringParameters;  // Called from web
+  } else {
+    params = event;  // Called from scheduler
+  }
```
Comment (mraerino) [inline entrypoint/fb.js]: This breaks if the http function is called without queryStringParameters. Should rather be determined by `event['detail-type'] == "Scheduled Event"`

Comment (mraerino) [reply entrypoint/fb.js]: More Info https://docs.aws.amazon.com/de_de/lambda/latest/dg/eventsources.html#eventsources-scheduled-event

Comment (mraerino) [reply entrypoint/fb.js]: Or some other method to determine if called via http

File: entrypoint/fb.js [hunk 3]
```diff
@@ -114,13 +113,11 @@ module.exports.message = (event, context, callback) => {
 module.exports.push = (event, context, callback = console.log) => {
   let params = null;
 
-  if (event.queryStringParameters !== undefined) {
-    params = event.queryStringParameters;  // Called from web
+  if ('timing' in event) {
+    params = event;
+  } else if (event.queryStringParameters && 'timing' in event.queryStringParameters) {
+    params = event.queryStringParameters;
```
Comment (mraerino) [inline entrypoint/fb.js]: Best to just use a `timing` variable instead of dragging other parameters with you

**Comment by mraerino** [comment]
We still need to figure out how to guarantee we are sending in CET/CEST times. This will need to go into this branch before we can merge.

**Comment by jh0ker** [comment]
merging after addressing all issues

**Pull request merged**